### PR TITLE
Add em(), rem(), remPair() and pxPair() utility functions

### PR DESCRIPTION
**Pull request by gregkohn**
Working simultaneously on reflecting this change in FedPack

### CODE REVIEW

File: utilities/fns/index.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const rem = (px) => `${px / 16}rem`
+export const remPair = (px) => ({ [px]: rem(px) })
+export const em = (px) => `${px / 16}em`
+export const pxPair = (px) => ({ [px]: `${px}px` })
```
Comment (mzlock) [inline utilities/fns/index.js]: These all assume a base font size of 16px. I prefer to work in multiples of 16 myself, but just making sure this is a good assumption before it's baked into a tool like this

**Review by mzlock — APPROVED** [review]

File: utilities/fns/index.js (hunk 1 above)
Comment (gregkohn) [reply utilities/fns/index.js]: Good point. Seems to me that making this assumption is a good opinion to lock in. Can anyone think of a good reason not to use 16 as the base?

Comment (mzlock) [reply utilities/fns/index.js]: Talking the design team into multiples of 16/8 has gone well so far. They seem to get it when it's infinitely divide-able and I think moooost of our sites have body copy that's 16px. It's not a bad assumption, buuuuut what happens in the one scenario where we might need to break it?

Comment (gregkohn) [reply utilities/fns/index.js]: Well, for example if this came up on FedPack (once it is updated to use these), I'd think you would not import the 16-based fns and write them yourself, as you do in FedPack currently.

File: utilities/fns/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# functions
+
+These helpers ease unit conversion and simplify config entries.
+
+- **em()**: Convert pixels into ems
```
Comment (jeremyfrank) [inline utilities/fns/README.md]: ~pixels~ pixel values
(to match the line below)

**Review by jeremyfrank — APPROVED** [review]
Nice update!

**Pull request merged**